diff --git a/dist/swagger-initializer.js b/dist/swagger-initializer.js
index 8ea0ea3a..b3ef27fe 100644
--- a/dist/swagger-initializer.js
+++ b/dist/swagger-initializer.js
@@ -3,7 +3,7 @@ window.onload = function() {
 
   // the following lines will be replaced by docker/configurator, when it runs in a docker-container
   window.ui = SwaggerUIBundle({
-    url: "https://petstore.swagger.io/v2/swagger.json",
+    url: "/dev/hippo-spec.yaml",
     dom_id: '#swagger-ui',
     deepLinking: true,
     presets: [
@@ -13,8 +13,8 @@ window.onload = function() {
     plugins: [
       SwaggerUIBundle.plugins.DownloadUrl
     ],
-    layout: "StandaloneLayout"
+    layout: "StandaloneLayout",
+    validatorUrl: "/swagger/validator"
   });
-
   //</editor-fold>
 };
